cmd/default-domain/main.go

```diff
@@ -226,5 +219,4 @@ func main() {
 	}
 
 	logger.Info("Updated default domain to: ", domain)
-	server.Shutdown(context.Background())
```

Comment:
Shutdown was blocking before and now it's not. This means main will exit and run `defer`'s etc.

Reply:
Ah yes, correct will fix.